### PR TITLE
Fix to navigate signatures when package names start with a single prefix

### PR DESCRIPTION
**Pull request by taeyeon-Kim**
resolved: #103

### CODE REVIEW

File: run-agent.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-./gradlew build -p scavenger-agent-java -x test
+./gradlew build -p scavenger-agent-java -x test -x integrationTest
```
Comment (taeyeon-Kim) [inline run-agent.sh:3]: integrationTest was slowing down the build, so it was excluded.

File: scavenger-frontend/src/components/snapshot/SnapshotDetail.vue [hunk 2]
```diff
@@ -328,6 +328,17 @@ export default {
...
+    isStartsWithSignature(querySignature, signature) {
+      if (!querySignature.startsWith(signature)) {
+        return false;
+      }
+
+      if (querySignature === signature) {
+        return true;
+      }
+
+      return querySignature.substring(signature.length).startsWith(".");
```
Comment (sohyun-ku) [inline scavenger-frontend/src/components/snapshot/SnapshotDetail.vue:340]: What is this line means?

File: scavenger-frontend/src/components/snapshot/SnapshotDetail.vue [hunk 3]
```diff
@@ -328,6 +328,17 @@ export default {
     resizeHorizontal(size) {
       localStorage.setItem("scavenger.snapshot.horizontal-size", size);
     },
+    isStartsWithSignature(querySignature, signature) {
+      if (!querySignature.startsWith(signature)) {
+        return false;
+      }
+
+      if (querySignature === signature) {
```
Comment (taeyeon-Kim) [inline scavenger-frontend/src/components/snapshot/SnapshotDetail.vue:336]: This condition is true when searching for a class.
```kotlin
      if (querySignature === signature) {
        return true;
      }
```

File: scavenger-frontend/src/components/snapshot/SnapshotDetail.vue (hunk 2 above)
Comment (taeyeon-Kim) [reply scavenger-frontend/src/components/snapshot/SnapshotDetail.vue:340]: The package must be followed by a class, so check for a `.` as well.

**Review by sohyun-ku — APPROVED** [review]

**Review by kojandy — APPROVED** [review]

**Pull request merged**